test/suite/test_async01.py

```diff
@@ -132,18 +132,9 @@ class test_async01(wttest.WiredTigerTestCase, suite_subprocess):
         ('table-row', dict(tablekind='row',uri='table')),
     ])
 
-    # Overrides WiredTigerTestCase so that we can configure
-    # async operations.
-    def setUpConnectionOpen(self, dir):
-        self.home = dir
-        conn_params = \
-                'create,error_prefix="%s: ",' % self.shortid() + \
+    conn_config = lambda self, dir: \
```

Comment:
@ddanderson I really like the simplifications this change provides, but I don't like Python lambda functions (mostly because I encounter them rarely enough that I need to go read documentation). Is there a different solution to setting up the conn_config?